README.md

```diff
@@ -1 +1,101 @@
-# shin_capstone
\ No newline at end of file
+# shin_capstone
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Microverse Capstone Project
+
+> This is a module 1: week 5 project in Microverse full stack course.
+
+
+## Built With
+
+- html
+- css
+- Javascript
+
+
+## Live Demo
+
+[**See Project 🚀**](https://strangeal.github.io/shin_capstone/)
```

Comment:
- [ ] Your deployed project link is not working and I really don't know the reason why 😞. This is what I can see when I checkout the link 👇 
![image](https://user-images.githubusercontent.com/90222110/190221784-48484ccd-72b7-4e23-93f0-bcade5202d98.png)

I would suggest that you deploy it again and make sure that the link is working very well 😉